packages/pds/src/db/index.ts

```diff
@@ -114,9 +163,23 @@ export default Database
 
 export type Dialect = 'pg' | 'sqlite'
 
+export type DialectFacets = PgFacets | SqliteFacets
```

Comment:
lmk if you have a better name than 'facets' for this. couldn't think of anything 😅

Reply:
Only idea I have is to call it something boring like `DbConfig`, and there's essentially a different config discriminated by `config.dialect`.